### DISCUSSION TITLE
General Question on Velocity Vectors

### DISCUSSION THREAD

**Discussion by jinyoung5484** [theislab/scvelo#739] [Q&A]
Thank you for this great tool!

I am a little confused about the differences between the velocities of cells and the values of cells plotted for the football-shaped plot of genes. I noticed that `scv.tl.velocity` function calculates velocities for each cell, but how do these velocities differ from the values that cells are plotted on the football-shaped plot for each gene?

Thanks!

**Comment by WeilerP** [comment]
@jinyoung5484, not entirely sure about your question: In a phase portrait each dot corresponds to a cell. You plot the abundance of unspliced RNA vs spliced RNA. The colouring is chosen by you. It is usually coloured by cell type or cluster, for example. This helps understands the temporal evolution of these clusters. Velocities themselves are a temporal change. You can, of course, also color cels in the phase portrait by their velocities.

**Reply by jinyoung5484** [reply]
Thank you for the response! 
That makes sense. Since cells are plotted on phase portraits of genes, does `scv.tl.velocity` calculate velocity for a cell by averaging the abundance of unspliced RNA vs spliced RNA of expressed genes?
